src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/Indication.java

```diff
@@ -54,6 +55,12 @@
     private String pattern;
 
     /**
+     * @deprecated, kept for backwards compatibility.
+     */
+    @Deprecated
+    private transient FailureCause cause;
+
```

Comment:
This doesn't make sense to me. The field only exists in one of your previous commits of this pull request. So your failure to deserialize should only happen when running an instance where you have ran one of your earlier commits, something that won't happen for anyone else.